README.md

```diff
@@ -1,2 +1,58 @@
 # publicly-available-microscopy-data
-Compiling and describing publicly available microscopy datasets
+## Purpose
+Compile and describe publicly available microscopy datasets using alpha and beta diversity measures. We extract metadata for each well and compute diveristy statistics for image attributes including but not limited to `screen id`, `genotype`, `phenotype`, `imaging method`, `cell line`, and `organism`. These measures give us insight into the quality of these datasets for use in image-based profiling methods.
+
+## Dependencies
+- python>=3.8
+- jupyterlab
+- pandas
+- plotnine
+- pyarrow
+- scipy
+
+### Conda Environment
+`cd` to your `publicly-available-microscopy-data` directory and run the following command:
+
+```bash
+conda env create -n microscopy-data
+```
+
+Activate the environment:
+
+```bash
+conda activate microscopy-data
+```
+
+## Statistics
```

Comment:
```suggestion
## Statistics

```

New line after H2 headings (all headings really)

Reply:
New lines after headings added.

Added in 1a10c619bd1a83be992e039f347a46aae6021a67.